admin/pages/api/links/[id].ts

```diff
@@ -0,0 +1,70 @@
+import type { NextApiRequest, NextApiResponse } from "next";
+import { getSession } from "next-auth/react";
+import { AliasedLink, mongoConnect } from "../../../utils";
+
+export default async function handler(
+  req: NextApiRequest,
+  res: NextApiResponse
+) {
+  const session = await getSession({ req });
+  switch (session?.user?.email) {
+    case process.env.ADMIN_EMAIL: {
```

Comment:
Yeah this looksss like it can be extended to work with memberDB, but I'll double check in the docs